0-ad-history-mod-package/simulation/templates/units/athen/hero_iphicrates.xml

```diff
@@ -8,6 +8,11 @@
     <GenericName>Iphicrates</GenericName>
     <SpecificName>Iphikratēs</SpecificName>
     <Icon>units/athen/hero_iphicrates.png</Icon>
+    <History>
+        Iphicrates was a notorious Athenian general in the 4th century BC. He had the reputation of being one of the most skilled commanders of his time, renown for his military successes, among others in the Corinthian War against Sparta. Most notably, though, Iphicrates is credited with reforming the Athenian military, primarily by developing a new type of light infantry unit. 
+        Compared to traditional hoplites, these new Iphicratean soldiers carried a longer spear and sword, and in place of the large, round shield, a smaller and lighter one. They also wore linen instead of the heavier bronze armor. Iphicrates even, being the son of a shoemaker, designed a new type of footwear called “Iphicratids”, that was easier to tie and untie. Besides these, “he [Iphicrates] also introduced many other useful improvements into warfare, but it would be tedious to write about them.” (Diod. 15.44)
+        While it is still a debated topic, soldiers in Iphicratean gear are likely to have fought as hoplites, meaning mainly as meelee units. However, because these Iphicratean hoplites were equipped more lightly than traditional hoplites, they were faster, more agile, and often able to outmaneuver the latter. They also proved to be useful as marines in the fleet, as when fighting onboard, agility and range are more crucial than maximum body protection.
```

Comment:
```suggestion
        While it is still a debated topic, soldiers in Iphicratean gear were likely to have fought as hoplites, meaning mainly as melee units. However, because these Iphicratean hoplites were equipped more lightly than traditional hoplites, they were faster, more agile, and often able to outmaneuver the latter. They also proved to be useful as marines in the Athenian fleet, because while fighting on ships, agility and range are more crucial than maximum body protection.```